src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/RecapitiPersonaFisicaTest.java

```diff
@@ -527,6 +543,17 @@ public void nellaPaginaITuoiRecapitiSiControllaCheLIndirizzoEmailNonSiaPresente(
         }
     }
 
+    @And("Nella pagina I Tuoi Recapiti si controlla che l'indirizzo Email non stata eleminata")
+    public void nellaPaginaITuoiRecapitiSiControllaCheLIndirizzoEmailNonStataEleminata() {
+        logger.info("Si controlla che l'indirizzo Email non stata eleminata");
+        if (recapitiDestinatarioPage.verificaMailAssociata()) {
+            logger.info("L'email non è stata eliminata");
+        } else {
+            logger.error("L'email è stata eliminata");
+            Assert.fail("L'email è stata eliminata");
+        }
+    }
+
```

Comment:
It seems similar to this: https://github.com/pagopa/pn-frontend-e2e/blob/4d0e2e2cc9224aa0d568f0da22892e280d12625d/src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/RecapitiPersonaFisicaTest.java#L537-L544. The behavior you're trying to check is almost the same. See if you can find a workaround.

Reply:
yes. one expects verificaMailAssociata to be true, the other to be false. One controls that email is present, the other that it is deleted. it is two different steps that uses one common method, I didnt get what exactly to change?